### DISCUSSION TITLE
Pixel 8 pro - can't install

### DISCUSSION THREAD

**Discussion by JHuberman** [OpenOrienteering/mapper#2187] [General]
My Pixel 8 pro said ”You can't install the app on your device" 

In my settings under "Install unknown apps" both"Files by Google" and"Chrome" are set to "Allow"

When I attempted to install OOM from Chrome it said the app was incompatible. 

Similar problem when attempting from "Files by Google"

Any suggestions?

**Comment by dg0yt** [comment]
Which APK file did you download?

**Comment by JHuberman** [comment]
Thanks for jumping in so quickly. I sure hope you can help me out

**Comment by JHuberman** [comment]
Kai,
I first want to confirm that you got the file name I sent you as a
screenshot.

Also are you making any progress on getting pixel 8 pro to install OOM?

Might this be a permissions issue on my phone? If so can you make any
suggestions?

Joseph Huberman
Raleigh NC
919-610-1788

**Reply by dg0yt** [reply]
I'm sorry, but attachements (incl. screenshots) are lost when you reply via e-mail. So my question is still without answer: Which APK file did you download?

I didn't make any other steps so far.

**Comment by JHuberman** [comment]
Sorry I didn't know you couldn't accept images. This is the file that I
sent...

0penOrienteering Mapper 0.9.5 Android armeabi v7a.apk

Joseph Huberman
Raleigh NC
919-610-1788

**Reply by dg0yt** [reply]
You should use the "OpenOrienteering-Mapper-0.9.5-Android-arm64-v8a.apk".
The other one is for older devices (32 bit).
Hope this helps.

**Comment by JHuberman** [comment]
Thanks for the suggestion to use a different file.  I had problems finding
the file, but finally succeeded.  It loaded on my Pixel8pro, and I tested
it and saw it working properly.

Thank you so much for your help, and more generally for keeping OOM working
so well.  It really has given me the confidence to be field checking our
orienteering maps for the Backwoods Orienteering Klub in NC.  I can focus
on *what *to map and how to show it rather than spending time on *where *it
is.

Thanks again for your help.
Joseph

================================
Joseph Huberman
904 Dorothea Drive
Raleigh, NC 27603

m. (919)610-1788

**Comment by dg0yt** [comment]
The 64-bit download is now the default for Android.
